components/mdc/TextInput/TextArea.svelte

```diff
@@ -53,22 +55,32 @@ const focus = async (node) => {
 label {
   width: 100%;
 }
+.counter {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  -webkit-font-smoothing: antialiased;
+  font-family: var(--mdc-typography-caption-font-family, var(--mdc-typography-font-family, Roboto, sans-serif));
+  font-size: var(--mdc-typography-caption-font-size, 0.75rem);
+  font-weight: var(--mdc-typography-caption-font-weight, 400);
+}
 </style>
 
 <label
   class="mdc-text-field mdc-text-field--outlined mdc-text-field--textarea {$$props.class} textfield-radius"
   class:mdc-text-field--no-label={!label}
   class:mdc-text-field--label-floating={label}
   class:mdc-text-field--with-internal-counter={maxlength}
-  class:mdc-text-field--invalid={hasReachedMaxLength}
   bind:this={element}
 >
   <textarea
     class="mdc-text-field__input"
     class:rtl
     aria-labelledby={labelID}
+    aria-controls="{labelID}-helper-id"
+    aria-describedby="{labelID}-helper-id"
     {rows}
-    {maxlength}
+    maxlength="524288"
```

Comment:
You may want to rename this component's "maxlength" just to avoid confusion with the HTML input's attribute called "maxlength".

Reply:
That's fine. I wasn't thinking about the consumer, just future developers!